### ISSUE TITLE
Breast mass is rubber-like in texture

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10069]
**Preferred term label:**
Breast mass is rubber-like in texture or not firm

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
https://www.ncbi.nlm.nih.gov/books/NBK551609/

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
added this modifier

Rubber-like lump
HP:0025765
def: This modifier is applied to a palpable mass or swelling in the body that has a rubber-like consistency, that is, it is soft and somewhat compressible.
